analyzer/codechecker_analyzer/analyzers/clangtidy/analyzer.py

```diff
@@ -324,7 +345,10 @@ def construct_analyzer_cmd(self, result_handler):
                     has_flag('--std', analyzer_cmd):
                 analyzer_cmd.append(self.buildaction.compiler_standard)
 
-            analyzer_cmd.extend(compiler_warnings)
+            if config.enable_all:
+                analyzer_cmd.append("-Weverything")
```

Comment:
This deserves some explanation in the code. Possibly in the summary.

Reply:
Yes this is a new development, I will include the explanation in the summary.